opencti-platform/opencti-graphql/src/manager/taskManager.js

```diff
@@ -155,13 +156,25 @@ const computeListTaskElements = async (context, user, task) => {
   const isUndefinedPosition = R.isNil(task_position) || R.isEmpty(task_position);
   const startIndex = isUndefinedPosition ? 0 : task_ids.findIndex((id) => task_position === id) + 1;
   const ids = R.take(MAX_TASK_ELEMENTS, task_ids.slice(startIndex));
+  let lastError;
   for (let indexId = 0; indexId < ids.length; indexId += 1) {
     const elementToResolve = ids[indexId];
     const element = await internalLoadById(context, user, elementToResolve, { type: DEFAULT_ALLOWED_TASK_ENTITY_TYPES });
     if (element) {
-      processingElements.push({ element, next: element.id });
+      try {
+        controlUserConfidenceAgainstElement(user, element);
+        processingElements.push({ element, next: element.id });
+      } catch (error) {
+        lastError = error;
+      }
     }
   }
+  if (lastError) {
+    await appendTaskErrors(
+      task,
+      [{ id: 'error--confidence-control', message: lastError.message }]
+    );
+  }
```

Comment:
@richard-julien I'm not sure about this. It will give just 1 error in the tasks error list, so the users knows something bad happened ; but not a zillion errors.